CHANGELOG.md

```diff
@@ -49,6 +49,10 @@
 -   [[user-storage]](#1.11.0_user-storage_scheme_updated) `UserStorageUri` scheme was changed from 'user_storage' to 'user-storage' as '\_' is not a valid char in scheme (according to [RFC 3986](https://tools.ietf.org/html/rfc3986#page-17)) [#9049](https://github.com/eclipse-theia/theia/pull/9049)
 
 
+## v1.11.0
+
+- [property-view] Added initial version of a selection-based property-view. A default implementation is available for file selections (via file navigator and default editors).
+
```

Comment:
minor: before merging we should place the following entry under the proper release `1.12.0`. You may also omit the entry, I usually complete the changelog before a release :+1: